starlite/openapi/controller.py

```diff
@@ -109,12 +154,25 @@ def retrieve_schema_json(self, request: Request) -> "OpenAPI":
                 A [Request][starlite.connection.Request] instance.
 
         Returns:
-            A rendered JSON object..
+            resoponse: With a rendered YAML object
```

Comment:
Multiple issues here - both a typo and `YAML` instead of `JSON`.

```suggestion
            A [Response][starlite.response.Response] instance with the JSON object rendered into a string.
```